.misc/deploy.pypi.sh

```diff
@@ -11,11 +11,15 @@ echo "Uploading coala to pypi"
 pip3 install twine wheel
 python3 setup.py sdist bdist_wheel
 # Upload one by one to avoid timeout
-# Don't block merging because pypi is in error 500 mood again
-set +e
-twine upload dist/*.whl -u "$PYPIUSER" -p "$PYPIPW"
-twine upload dist/*.tar.gz -u "$PYPIUSER" -p "$PYPIPW"
-set -e
+twine upload dist/* -u "$PYPIUSER" -p "$PYPIPW" 2>&1 | tee twine_output.txt
+if [ "${PIPESTATUS[0]}" -nq 0 ]; then
```

Comment:
Sorry, I made this accidental error while changing the name of the output.txt.
It was `-ne` in my previous commits.
I've tested it on a local repo, but I've not seen the `server error 500` yet.